plugins/repoquery.py

```diff
@@ -222,6 +230,39 @@ def configure(self, args):
         demands.sack_activation = True
         demands.available_repos = True
 
+    def by_all_deps(self, name, query):
+        defaultquery = query.filter(name=name)
+        requiresquery = self.by_requires(self.base.sack, name, query)
+        allpkgs = []
+        for pkg in defaultquery:
+            for reqpkg in requiresquery.run():
+                allpkgs.append(reqpkg)
+
+            # for filePath in pkg.files:
```

Comment:
Update: it is not actually desired to search for all packages requiring files of the package. Delete it, please, it should have the whole `--alldeps` functionality now.